### DISCUSSION TITLE
cheapest laptop?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#874] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
title, preferably not chromebook

**Comment by Hilololsusy** [comment]
look on amazon refurbished laptop boom

**Reply by Hilololsusy** [reply]
Also just use linux os after buying a chromebook, gl finding something under 150 using windows

**Comment by rwg100** [comment]
cant help with a laptop but school are throwing away old dell optiplexes with have decent specs like 4th gen i7s and 16gb or 32gb of ddr3 and like a 256gb m.2

**Reply by rwg100** [reply]
and they cost like 90 dollars

**Comment by 3kh0** [comment, marked as answer]
![image](https://github.com/3kh0/ext-remover/assets/58097612/9e769a29-38e1-4895-89cd-3805a315e3a5)

https://www.ebay.com/itm/176147956116?epid=14041880715&hash=item29033c8194:g:QokAAOSwTYNlk9Sx

Note that is for a gaming laptop which has a dedicated graphics card. If you need something cheaper try something from HP

![image](https://github.com/3kh0/ext-remover/assets/58097612/bd4b28dc-94cb-4fd4-8b30-d073287d5626)

https://www.ebay.com/itm/375167583184?hash=item5759bb03d0:g:19sAAOSwsAJll~cL

**Reply by rwg100** [reply]
number 2 has no os most likely because desc says its booths straight to bios also stay away from buying from HP because their laptops have the shittiest quality of all time buy something like an acer or dell

**Reply by rwg100** [reply]
also stay away from hdds and 8gb of ram
